api/lib/domain/services/authentication/oidc-authentication-service.js

```diff
@@ -21,54 +16,59 @@ import { OIDC_ERRORS } from '../../constants.js';
 import { temporaryStorage } from '../../../infrastructure/temporary-storage/index.js';
 
 const DEFAULT_REQUIRED_PROPERTIES = ['clientId', 'clientSecret', 'authenticationUrl', 'userInfoUrl', 'tokenUrl'];
-
+const DEFAULT_SCOPE = 'openid profile';
 const DEFAULT_REQUIRED_CLAIMS = ['sub', 'family_name', 'given_name'];
 
 const defaultSessionTemporaryStorage = temporaryStorage.withPrefix('oidc-session:');
 
 class OidcAuthenticationService {
   #isReady = false;
   #isReadyForPixAdmin = false;
-
   #requiredClaims = Array.from(DEFAULT_REQUIRED_CLAIMS);
 
   constructor(
     {
-      identityProvider,
+      additionalRequiredProperties,
+      authenticationUrl,
+      extraAuthorizationUrlParameters,
+      claimsToStore,
+      clientId,
+      clientSecret,
       configKey,
-      source,
-      slug,
-      organizationName,
+      endSessionUrl,
       hasLogoutUrl = false,
+      identityProvider,
       jwtOptions,
-      clientSecret,
-      clientId,
+      organizationName,
+      postLogoutRedirectUri,
+      redirectUri,
+      scope = DEFAULT_SCOPE,
+      slug,
+      source,
       tokenUrl,
-      authenticationUrl,
-      authenticationUrlParameters,
       userInfoUrl,
-      endSessionUrl,
-      postLogoutRedirectUri,
-      additionalRequiredProperties,
-      claimsToStore,
+      wellKnownUrl,
```

Comment:
À la réflexion à la place de `wellKnownUrl` je préférerais qu'on utilise `openidConfigurationUrl` : 
```suggestion
      openidConfigurationUrl,
```

Je suis désolé, cela a déjà été soulevé une fois par @igorissen, mais je me suis rendu compte en expliquant aux uns et aux autres que c'est vraiment la notion de « openid-configuration » dont il s'agit. Alors je sais ça fait beaucoup de changements, mais je suis prêt à les faire.